pkgs/applications/networking/instant-messengers/discord-screenaudio/default.nix

```diff
@@ -0,0 +1,71 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, makeShellWrapper
+, wrapQtAppsHook
+, cmake
+, qtbase
+, qtwebengine
+, knotifications
+, kxmlgui
+, kglobalaccel
+, pipewire
+, bash
+, xdg-desktop-portal
+}:
+
+stdenv.mkDerivation rec {
+  pname = "discord-screenaudio";
+  version = "1.9.2";
+
+  src = fetchFromGitHub {
+    owner = "maltejur";
+    repo = "discord-screenaudio";
+    rev = "v${version}";
+    sha256 = "sha256-it7JSmiDz3k1j+qEZrrNhyAuoixiQuiEbXac7lbJmko=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    makeShellWrapper
+    wrapQtAppsHook
+    cmake
+    qtbase
+    qtwebengine
+    knotifications
+    kxmlgui
```

Comment:
These should probably be in buildInputs add a `strictDeps = true` above `nativeBuildInputs`